src/ASM/LR/ASMu3Dmx.C

```diff
@@ -769,3 +769,18 @@ Vec3 ASMu3Dmx::getCoord (size_t inod) const
   }
   return Vec3(&(*basis->cp()),nsd);
 }
+
+
+void ASMu3Dmx::remapErrors(Vector& errors, const Vector& origErr) const
```

Comment:
You don't really use the algebra operations on these argument, so it makes more sense to use RealArray (which is an alias for `std::vector<double>`) as argument type instead in this method

Reply:
changed.